server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -234,6 +232,13 @@ public abstract class BlobStoreRepository extends AbstractLifecycleComponent imp
      */
     public static final Setting<Boolean> COMPRESS_SETTING = Setting.boolSetting("compress", true, Setting.Property.NodeScope);
 
+    /**
+     * Size hint for the IO buffer size to use when reading from and writing to the repository.
+     */
+    private static final Setting<ByteSizeValue> IO_BUFFER_SIZE_SETTING = Setting.byteSizeSetting("io_buffer_size",
```

Comment:
Had to rename it in https://github.com/crate/crate/pull/13313/commits/baa34a1e74ae168840ec0f2f5dfb2c56a485a48f
as we already have `public static BUFFER_SIZE_SETTING` in `S3RepositorySettings` and those vars got messed up